pyrightconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extraPaths": [
+    "rerun_py/rerun_sdk"
+  ],
+
+  "exclude": [
+    "**/node_modules",
+    "**/__pycache__"
+  ],
+
+  "ignore": [
+    "rerun_py/rerun",
+    "rerun_py/rerun_demo"
+  ],
+
+  "defineConstant": {
+    "DEBUG": true
+  },
+
+  "venv": "venv",
+
+  "reportPrivateImportUsage": false
+}
```

Comment:
missing newline at the end of the file - tip: configure your editor to always insert a newline at the end of each file, and to trim trailing whitespace from every line

Reply:
I was convinced I had that setting but indeed it was disabled—fixed now.